tests/logic.test.js

```diff
@@ -0,0 +1,6 @@
+const tape = require('tape');
+
+tape('dummyTest', (t) => {
```

Comment:
Just a small change to do here, I'm not crazy about the naming of this test, doesn't look so clean to name it `dummyTest`. Can we change it for the more conventional `Tape is working` statement. Looks cleaner and its easier to understand why this is there.